pkg/trace/test/testutil/stats.go

```diff
@@ -6,22 +6,35 @@
 package testutil
 
 import (
+	"github.com/DataDog/datadog-agent/pkg/trace/pb"
 	"github.com/DataDog/datadog-agent/pkg/trace/stats"
 )
 
-const defaultEnv = "none"
+const (
+	defaultEnv      = "none"
+	defaultHostname = "hostname"
+)
 
 // BucketWithSpans returns a stats bucket populated with spans stats
-func BucketWithSpans(spans []*stats.WeightedSpan) stats.Bucket {
+func BucketWithSpans(spans []*stats.WeightedSpan) pb.ClientStatsBucket {
 	srb := stats.NewRawBucket(0, 1e9)
 	for _, s := range spans {
-		srb.HandleSpan(s, defaultEnv)
+		// override version to ensure all buckets will have the same payload key.
+		s.Meta["version"] = ""
+		srb.HandleSpan(s, defaultEnv, defaultHostname)
+	}
+	buckets := srb.Export()
+	if len(buckets) != 1 {
+		panic("All entries must have the same payload key.")
```

Comment:
This seems risky 🙂 Do we really want to have a full crash and panic here?

Reply:
This is in `testutil`, do you think it's an issue to panic during tests?

Reply:
You're right. Sorry I missed that. Resolving.